client/app/layout/Layout.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+export default class Layout extends React.Component {
+
+  static propTypes = {
+    children: PropTypes.object.isRequired,
+  }
+
+  render() {
+    return (
+      <section>
+        <header>
+          <ul>
+            <li>
+              <Link to="/">Comments</Link>
```

Comment:
These will not automatically highlight. 

We need set the activeClassName property to true on the current link.

For an example of how to do this, react-router docs.